package.json

```diff
@@ -28,54 +28,69 @@
   },
   "homepage": "https://github.com/Microsoft/PowerBI-visuals-WordCloud#readme",
   "dependencies": {
-    "d3": "5.12.0",
-    "jquery": "3.4.1",
-    "lodash": "4.17.15",
-    "powerbi-visuals-utils-colorutils": "^2.2.1",
-    "powerbi-visuals-utils-dataviewutils": "^2.2.1",
-    "powerbi-visuals-utils-formattingutils": "^4.4.2",
-    "powerbi-visuals-utils-svgutils": "^2.2.1",
-    "powerbi-visuals-utils-tooltiputils": "^2.3.1",
-    "powerbi-visuals-utils-typeutils": "^2.2.1",
-    "powerbi-models": "^1.0.2"
+    "d3": "7.2.1",
```

Comment:
Package.json shouldn't contain the standalone version of D3. Replace it with smaller d3 packages.